import_sdrf_files.R

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env Rscript 
+
+### Import SDRF files for the downloaded classifiers
+### By default, SDRF files for all available datasets that have trained classifiers are obtained
+### Alternatively, necessary datasets can be provided through a config file
+
+option_list = list(
+    make_option(
+            c("-f", "--config-file"),
+            action = "store",
+            default = NA,
+            type = 'character',
+            help = "Config file in .yaml format"
+    ),
+    make_option(
+            c("-k", "--get-condensed-sdrf"),
+            action = "store_true",
+            default = FALSE,
+            type = 'logical',
+            help = "Should condensed SDRF file(s) be downloaded? Default: FALSE"
+    ),
+    make_option(
+            c("-s", "--sdrf-output-dir"),
+            action = "store_true",
+            default = NA,
+            type = 'character',
+            help = "Output path for imported SDRF files directory"
+    )
+)
+opt = wsc_parse_args(option_list, mandatory = c("tool", "classifiers_output_dir"))
+# import dependencies 
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+suppressPackageStartupMessages(require(R.utils))
+suppressPackageStartupMessages(require(yaml))
+suppressPackageStartupMessages(require(RCurl))
+
+# parse config file or use default values
+if(!is.na(opt$config_file)){
+    config = yaml.load_file(opt$config_file)
+    datasets = toupper(config$datasets)
+    scxa_experiments_prefix = config$scxa_experiments_prefix
+    if(!endsWith(scxa_experiments_prefix, "/")) scxa_experiments_prefix = paste(scxa_experiments_prefix, "/", sep="")
+} else {
+    scxa_experiments_prefix = "ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/sc_experiments/"
```

Comment:
Could you put the default URIs in some config file so they're not hard-coded anywhere? I'm seeing the same path in a few places which is a recipe for maintenance issues ;-)

Reply:
@pinin4fjords but there is already an option - either use user-provided prefixes from yaml config or use default ones

Reply:
Minor doubt @a-solovyev12, what's the difference between `action = "store"` and ` action = "store_true"` ?

Reply:
@rubenchazarra, `action=store_true` is used specifically for Boolean variables.